cmd/validate/image_test.go

```diff
@@ -122,6 +123,39 @@ func Test_determineInputSpec(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "ApplicationSnapshot YAML string",
+			arguments: data{
+				input: hd.Doc(`
+                ---
+                application: app1
+                components:
+                - name: nodejs
+                  containerImage: quay.io/hacbs-contract-demo/single-nodejs-app:877418e
+                - name: petclinic
+                  containerImage: quay.io/hacbs-contract-demo/spring-petclinic:dc80a7f
+                - name: single-container-app
+                  containerImage: quay.io/hacbs-contract-demo/single-container-app:62c06bf
+                `),
```

Comment:
Indent looks off, but I'm guessing it's spaces instead of tabs. Consider putting real tabs in maybe?

Reply:
Tabs are not allowed in YAML.

Reply:
Oh, I did not know that..

Reply:
Well.. it turns out that `hd.Doc` does remove the leading tabs. Updated.

Reply:
Ah yes. The unusual case where some tabs followed by some spaces is correct. :smile: .